e2e/tests/components/menu/menu-item-focus/menu-item-focus.e2e-spec.ts

```diff
@@ -0,0 +1,152 @@
+import { browser, Key } from 'protractor';
+import { imageCompare } from '../../common/image-compare';
+import { MenuItemFocusPage } from './menu-item-focus.po.spec';
+
+describe('Menu Item Focus Directive Tests', () => {
+
+    let page: MenuItemFocusPage;
+
+    beforeEach(async () => {
+        page = new MenuItemFocusPage();
+        await page.getPage();
+    });
+
+    it('should open the and focus the component input', async () => {
+        await page.topFocus.click();
+
+        await browser.actions()
+            .sendKeys(Key.TAB)
+            .sendKeys(Key.SPACE)
+            .perform();
```

Comment:
A lot of the tests do this as the first step, you can refactor this using describe and beforeEach.